src/ExpenseTracker.Common/Model/BaseModel.cs

```diff
@@ -1,12 +1,8 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
 namespace ExpenseTracker.Common.Model
 {
     public class BaseModel
     {
-        public virtual long Id { get; set; }
+        public long Id { get; set; }
+        public string Status { get; set; } = "Active";
```

Comment:
@BirajMainali Add constants for Active and Inactive and use that constant here.

Reply:
I will change this on the workspace trash issue.

Reply:
@BirajMainali We should do such things in the PR itself. Allowing it for now 😂😂